ratis-common/src/main/java/org/apache/ratis/util/ProtoUtils.java

```diff
@@ -118,6 +119,7 @@ static RaftPeer toRaftPeer(RaftPeerProto p) {
         .setClientAddress(p.getClientAddress())
         .setAdminAddress(p.getAdminAddress())
         .setPriority(p.getPriority())
+        .setStartupRole(p.hasStartupRole() ? p.getStartupRole() : RaftProtos.RaftPeerRole.FOLLOWER)
```

Comment:
Since the default value is set above, redundant judgment logic can be deleted here

Reply:
This is to ensure forward compatibility. Using the previous raft log will cause startupRole item to not exist.

Reply:
Good Catch! I forget the forward compatibility